.github/workflows/main.yml

```diff
@@ -53,6 +53,15 @@ jobs:
       - name: go compile
         run: make build
 
+      - name: gomobile install
+        run: go install golang.org/x/mobile/cmd/gomobile@latest
+
+      - name: gomobile init
+        run: gomobile init
+
+      - name: android compile
+        run: make android
```

Comment:
The CI seems to fail with `no exported names in the package "github.com/lightninglabs/lightning-node-connect/mobile"`. I don't have either of the SDKs installed, so can't do a local debug.